### ISSUE TITLE
Build is failing with wsdl file issue

### ISSUE THREAD

**Issue by ghost** [YetAnotherSideProject/MaStR-Fetcher#1]
Below is the issue during the mvn clean install 

<img width="1509" alt="Screenshot 2023-03-15 at 11 03 52 PM" src="https://user-images.githubusercontent.com/64338218/225395179-66c0a7e3-aa44-4ad8-a0a6-e1d8f8702e2b.png">

<img width="1512" alt="Screenshot 2023-03-15 at 11 09 09 PM" src="https://user-images.githubusercontent.com/64338218/225396140-85ac7e04-67a4-4804-b538-bca89d38e368.png">

Also, I have the following configs in my project and facing the same problem.

Spring Boot  : 3.0.4
JDK :   17.0.6.10.1

pom config :

 <groupId>com.helger.maven</groupId>
 <artifactId>jaxb40-maven-plugin</artifactId>
 <version>0.16.1</version>

But the error I have faced.

<img width="1293" alt="Screenshot 2023-03-15 at 11 12 26 PM" src="https://user-images.githubusercontent.com/64338218/225397759-c8b65277-b88e-424e-97c5-e1b2899a4b94.png">
<img width="959" alt="Screenshot 2023-03-15 at 11 12 36 PM" src="https://user-images.githubusercontent.com/64338218/225397768-1e19345e-5f6c-48ce-8c6f-42f9e99e683c.png">
<img width="1188" alt="Screenshot 2023-03-15 at 11 13 19 PM" src="https://user-images.githubusercontent.com/64338218/225398538-b108771a-b131-43b0-9c0d-4edb0aca2b15.png">

<img width="1195" alt="Screenshot 2023-03-15 at 11 13 04 PM" src="https://user-images.githubusercontent.com/64338218/225397787-47a03ce5-7c97-4121-85c8-bd9c046c6c0d.png">
<img width="1188" alt="Screenshot 2023-03-15 at 11 13 19 PM" src="https://user-images.githubusercontent.com/64338218/225397801-f139b182-071b-4415-8e9b-b2f7d958bb35.png">

Could you please explain why these errors are popping up? and should I have updated any version?
Or these errors are negligible?

**Comment by ghost** [comment]
@Mattizin could you please clarify those?

**Comment by Mattizin** [comment]
The use of JAXB (Java XML Binding) to compile web service definitions into Java classes is really confusing now that the JAXB components have been separated from the core JDK itself.
There's a lot of outdated information out there, and multiple forks & versions trying to fit individual use cases.
The way implemented here worked for the current scope of this hobby project, I just ignored the warnings.

As the *.wsdl compilation is experimental, and the MaStR web service provides the *.xsd files, I switched to compiling them, and adjusted the individual binding configuration specifically to the needs of the service objects.
Now the compilation process works without any errors & warnings on my end, so I hope the commit (https://github.com/YetAnotherSideProject/MaStR-Fetcher/commit/f4b440e48b6899788c1f2eb0c83286df3e4e601f) solves your question.
